includes/admin/class-wc-stripe-settings-controller.php

```diff
@@ -81,6 +81,11 @@ public function admin_scripts() {
 		];
 		wp_localize_script( 'woocommerce_stripe_admin', 'wc_stripe_settings_params', $params );
 
+		$settings = [
+			'accountStatus' => WC_Stripe_Account::get_account_status(),
+		];
+		wp_localize_script( 'woocommerce_stripe_admin', 'wc_stripe_admin_settings', $settings );
```

Comment:
Out of curiosity - the `woocommerce_stripe_admin` is already part of `wp_localize_script` a few lines above - why add a new global `wc_stripe_admin_settings`, could we have leveraged the existing `wc_stripe_settings_params`?

Reply:
I was thinking of keeping the saved settings related values in one variable ( `wc_stripe_admin_settings` ) and the other params (flags and all) in the previous one. We can have all in the same variable though. Does not make much differences.